storage/client/src/main/java/com/azure/storage/blob/AppendBlobClientBuilder.java

```diff
@@ -37,16 +37,16 @@
  * Fluent AppendBlobClientBuilder for instantiating a {@link AppendBlobClient} or {@link AppendBlobAsyncClient}.
  *
  * <p>
- * An instance of this builder may only be created from static method {@link AppendBlobClient#appendBlobClientBuilder()}.
- * The following information must be provided on this builder:
+ * An instance of this pageBlobClientBuilder may only be created from static method {@link AppendBlobClient#appendBlobClientBuilder()}.
+ * The following information must be provided on this pageBlobClientBuilder:
```

Comment:
It appears there has been a global 'pageBlobClientBuilder' rename that has accidentally touched the javadoc. I won't comment any further on this - but it looks like they should all be reverted.